apps/server/src/api-data/rundown/rundown.service.ts

```diff
@@ -107,7 +108,7 @@ export async function editEntry(patch: PatchWithId): Promise<OntimeEntry> {
     updateRuntimeOnChange(rundownMetadata);
 
     // notify timer and external services of change
-    notifyChanges(rundownMetadata, revision, { timer: [entry.id], external: true });
+    notifyChanges(rundownMetadata, revision, { timer: didInvalidate ? true : [entry.id], external: true });
```

Comment:
question: when we send true, we invalidate the entire rundown, is that necessary?

Reply:
is it not? if we get `didInvalidate` then the `rundownMutation.edit` had to change something not in our `RegenerateWhitelist` that in case off linked timers means a lot of stuff could have changed.
What changed would then only be known by the `commit` function, but that is currently not collecting that data